python/lsst/pipe/tasks/hips.py

```diff
@@ -489,6 +531,8 @@ def build_quantum_graph(
             Collection or collections to search for input datasets, in order.
             If not provided, ``registry.defaults.collections`` will be
             searched.
+        butler_config : `str`, optional
```

Comment:
This is not a parameter. Should be `metadata`?

Reply:
Right, I tried first to instantiate metadata in this method, but it required a bunch of extra parameters, so I decided just pass metadata here as an argument (but forgot to clean the docstring). Now I'm also dropping `output_run` and using one from metadata.